packages/storage-plugin/tests/storage.plugin.spec.ts

```diff
@@ -218,29 +219,13 @@ describe('NgxsStoragePlugin', () => {
         }
       };
 
-      get length() {
```

Comment:
You can not delete detail test

Reply:
It was to be consistent with the StorageEngine API.
I will add it back when the comment below is resolved.